### ISSUE TITLE
Addon folderlistmenu  partialy broken

### ISSUE THREAD

**Issue by herbrot** [tablacus/TablacusExplorerAddons#383]
Entries with Grave accent (`) do not work anymore
```
#EXTIMG:C:\Windows\System32\notepad.exe
Edit `notepad %selected%`
```
Since I use these functionality rarely I just noticed it a few days before.
I found the last working release was 240227

(as workaround I use javascript:command(wsh.run('prog "arg  .."')) which works unless ther is a '?' in URLs)
Since this is not a big issue maybe just change the documentation to reflect the behavior

**Comment by tablacus** [comment]
I have fixed it on Folder list menu v1.16.
Please try to update it.

Regards,

**Comment by herbrot** [comment]
wow that was really fast!
thank you very much

Best Regards
